lib/widgets/autocomplete.dart

```diff
@@ -247,6 +247,50 @@ class _MentionAutocompleteItem extends StatelessWidget {
   }
 }
 
+class _EmojiAutocompleteItem extends StatelessWidget {
+  const _EmojiAutocompleteItem({required this.option});
+
+  final EmojiAutocompleteResult option;
+
+  static const _size = 32.0;
+  static const _notoColorEmojiTextSize = 25.7;
+
+  @override
+  Widget build(BuildContext context) {
+    final store = PerAccountStoreWidget.of(context);
+    final candidate = option.candidate;
+
+    final emojiDisplay = candidate.emojiDisplay.resolve(store.userSettings);
+    final Widget? glyph = switch (emojiDisplay) {
+      ImageEmojiDisplay() =>
+        ImageEmojiWidget(size: _size, emojiDisplay: emojiDisplay),
+      UnicodeEmojiDisplay() =>
+        UnicodeEmojiWidget(
+          size: _size, notoColorEmojiTextSize: _notoColorEmojiTextSize,
+          emojiDisplay: emojiDisplay),
+      TextEmojiDisplay() => null, // The text is already shown separately.
```

Comment:
It's true that the text is shown, but in a way that's more confusing than it needs to be, I think:

<img width="675" alt="image" src="https://github.com/user-attachments/assets/03f5c1b0-0922-48d4-8f23-c28d969b136b">

I'll tap "person_tipping_hand" if it looks appealing, but then I might be confused/unhappy to see ":information_desk_person:" written instead.

Is there a nice way to give the "emoji name" some distinct formatting? Maybe borrowing the label/sublabel design from the Figma on @-mention autocomplete:

<img width="757" alt="image" src="https://github.com/user-attachments/assets/06089aa6-5458-4765-ba8b-4a4d8cbd11a7">

where the first line says ":information_desk_person:" (or maybe without the colons) and the second says "person_tipping_hand" (plus more if the list is longer)? That same design might be helpful for unicode- and image-display emoji too (though less necessary), so the code could just cleanly apply it unconditionally. This would grow some of the items vertically (splitting what could be one line of text into two), but I think that's an OK tradeoff.

Also I could imagine someone using the "plain text" setting because they don't like seeing emoji everywhere, but they still want to see the emoji in the emoji picker, so they know how it'll look to people who don't use the setting.

Reply:
Sure, that all sounds fine.